docs/requirements.txt

```diff
@@ -21,3 +21,4 @@ torchvision==0.9.1+cpu
 cnest==1.0.4
 gym3==0.3.3
 procgen==0.10.4
+protobuf==3.20.*
```

Comment:
Not sure why this is a "downgrade". seems to me like "adding protobuf", no?

Reply:
> Not sure why this is a "downgrade". seems to me like "adding protobuf", no?

protobuf is included in autodoc. Somehow RTD upgrades its version recently which caused the issue.

Reply:
Gotcha. Thanks for the clarification.